api/server/v1/validator.go

```diff
@@ -103,6 +103,10 @@ func (x *SearchRequest) Validate() error {
 		return err
 	}
 
+	if len(x.IncludeFields) > 0 && len(x.ExcludeFields) > 0 {
+		return Errorf(Code_INVALID_ARGUMENT, "Cannot use both `include_fields` and `exclude_fields` together")
+	}
```

Comment:
What is the behavior when none of them set?

Reply:
No field projection to apply, this particular code path will be a no-op and all document fields will be returned

Reply:
got it.